website/docs/reference/global-configs/behavior-changes.md

```diff
@@ -164,3 +166,12 @@ In previous versions (dbt Core 1.8 and earlier), the MetricFlow time spine confi
 When the flag is set to `True`, dbt will continue to support the SQL file configuration. When the flag is set to `False`, dbt will raise a deprecation warning if it detects a MetricFlow time spine configured in a SQL file. 
 
 The MetricFlow YAML file should have the `time_spine:` field. Refer to [MetricFlow timespine](/docs/build/metricflow-time-spine) for more details. 
+
+### Custom microbatch strategy
+The `require_batched_execution_for_custom_microbatch_strategy` flag is set to `False` by default and is only relevant if you already have a custom microbatch macro in your project.  If you don't have a custom microbatch macro, you don't need to set this flag as dbt will handle microbatching automatically for any model using the [microbatch strategy](#how-microbatch-compares-to-other-incremental-strategies).
```

Comment:
Hiya @mirnawong1 

The microbatch link didn't direct me to a page. I _think_ the page should be /docs/build/incremental-microbatch#how-microbatch-compares-to-other-incremental-strategies

Kind Regards
Natalie

Reply:
great catch!